BaseFilter/sections/antiadblock.txt

```diff
@@ -831,6 +831,7 @@ starwarsaddicted.it,wuyong.fun,26g.me,myqqjd.com,poapan.xyz,javboys.com,idevicec
 ##a[id][href="https://toolkitspro.com"][rel="noopener noreferrer"]
 ##a[id][href="https://chpadblock.com/"][rel="noopener noreferrer"]
 ! specific
+recipenp.com#%#//scriptlet('prevent-eval-if', '/fairAdblock|chp_?adblock|adsbygoogle\.js/')

```

Comment:
It's `chp_ads_blocker`. We have rules for this.
Find this section and some of the scriptlets should help.
However, there is section for content blockers that do not support such rules, but you can find an existing rule that works fine there.